modules/nextflow/src/main/groovy/nextflow/cli/Launcher.groovy

```diff
@@ -209,7 +209,7 @@ class Launcher {
 
             // when the first argument is a file, it's supposed to be a script to be executed
             if( i==1 && !allCommands.find { it.name == current } && new File(current).isFile()  ) {
-                normalized.add(0,CmdRun.NAME)
+                normalized.add(0,CmdRun.NAME.toString())
```

Comment:
Why is this needed when CmdRun.NAME is already declared as a String?

Reply:
Don't remember